src/service/variable-source.js

```diff
@@ -165,6 +168,7 @@ export class VariableSource {
         this.replacements_[varName] || {sync: undefined, async: undefined};
     this.replacements_[varName].sync = syncResolver;
     this.replacementExpr_ = undefined;
+    this.replacementExprWithoutArgs_ = undefined;
```

Comment:
maybe name it `replacementExprV2_`, so we know the other one is going to be remove after v2 launch.

in general, we can have consistent naming (suffix of v2) so the later clean up can be easier.